pkg/service/backtest.go

```diff
@@ -138,25 +158,40 @@ func (s *BacktestService) QueryKLinesBackward(exchange types.ExchangeName, symbo
 }
 
 func (s *BacktestService) QueryKLinesCh(since, until time.Time, exchange types.Exchange, symbols []string, intervals []types.Interval) (chan types.KLine, chan error) {
-	sql := "SELECT * FROM `binance_klines` WHERE `end_time` BETWEEN :since AND :until AND `symbol` IN (:symbols) AND `interval` IN (:intervals) ORDER BY end_time ASC"
-	sql = strings.ReplaceAll(sql, "binance_klines", exchange.Name().String()+"_klines")
+
+	if len(symbols) == 0 {
+
+		log.Error("backtest: symbols is empty when querying kline, plesae check your strategy setting. ")
```

Comment:
errC is sending the same error message, so this one can be removed?

Reply:
agree